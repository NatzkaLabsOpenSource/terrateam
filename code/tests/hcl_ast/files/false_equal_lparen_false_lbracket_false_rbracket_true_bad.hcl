foo = (foo[foo] true
---
foo = (foo[foo]foo
