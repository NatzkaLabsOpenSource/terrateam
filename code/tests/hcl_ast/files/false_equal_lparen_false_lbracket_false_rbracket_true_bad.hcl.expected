Unexpected expression found after index operator.
